validation-test/stdlib/SequenceType.swift.gyb

```diff
@@ -617,6 +617,26 @@ SequenceTypeTests.test("contains/Predicate") {
 }
 
 //===----------------------------------------------------------------------===//
+// first()
+//===----------------------------------------------------------------------===//
+
+SequenceTypeTests.test("first/Predicate") {
```

Comment:
Sorry, it is not documented.  The general principle is that we want to put all algorithm tests into the reusable `StdlibCollectionUnittest` module, so that we can maximize our algorithm and collection coverage.